### DISCUSSION TITLE
Assign Catalog to Category Afterwards

### DISCUSSION THREAD

**Discussion by marcus-at-localhost** [atrocore/atrocore#422] [Q&A] [answered]
Hi, I stumble upon the following problem, every time I need a new category.

For example (using the demo instance)

1.  Goto https://demo.atropim.com/#Category 
2. Create Category 
3. Select Parent Category "Fashion"
4. Save new "For Children" category
5. Can't Assign "Catalogs"
6. ![image](https://user-images.githubusercontent.com/1337223/183419206-3755ceaa-c2de-4283-8d2a-b2cd227996ec.png)
7. Goto Catalog/Fashion https://demo.atropim.com/#Catalog/view/628b3cfe389794ea4
8. Assign Category - Nothing to select :-( ![image](https://user-images.githubusercontent.com/1337223/183419530-d8dae8ff-4cee-4897-b3d1-d365bb448030.png)

What works is creating a new category in Catalogs 
![image](https://user-images.githubusercontent.com/1337223/183419766-693e8fad-92bb-46b7-bdd6-1e0e5e16aaae.png)
and then everything works as expected.

But why is it not possible to do it the more "natural way (in my opinion)" and create the Category in int Category entity and get the catalog automatically assigned by the Parent category I create that in?

Is that a bug or a feature that I don't understand?

**Comment by rratsun** [comment]
Hello Marcus,

Only root category can be linked with catalog. That's why point 5 and 8 happened. But the problem exist. I expect that new category should be in catalog because you create new category with parent. We will check it soon and fix this bug if it needs.

**Comment by rratsun** [comment, marked as answer]
Fixed at PIM 1.5.55

**Reply by marcus-at-localhost** [reply]
Works as expected. Thank you so much!
